src/main/java/UserInterface.java

```diff
@@ -47,8 +41,8 @@ public void printWelcomeMessage() {
         print(WELCOME);
     }
 
-    public String printPlayersMenu() {
-        print(PLAYERSMENU);
+    public String printMenuAndGetOption(String menu) {
+        print(menu);
         return console.read();
```

Comment:
Can't think of a way of splitting it, since both lines of code are related. Maybe it's better to just rename it to "`getMenuOption()`"?